src/core/manager.c

```diff
@@ -3522,6 +3551,12 @@ static void manager_notify_finished(Manager *m) {
                                                FORMAT_TIMESPAN(userspace_usec, USEC_PER_MSEC),
                                                FORMAT_TIMESPAN(total_usec, USEC_PER_MSEC)));
                 }
+
+                /* Send SD_READY=1 to hypervisor. Note that we don't check that we are running in a VM, nor that the passed
+                 * notify_socket credential is a VSOCK address. While that's the documented use case, it doesn't hurt to let
+                 * people try weirder things if they want to, after all setting the credential requires privileges over pid1
+                 * so there's no reason not to. */
+                send_notify_ready_via_creds();
```

Comment:
pid1 already issues sd_notify() at the right places, see manager_print_jobs_in_progress(), manager_send_ready() and thelike. I think we should just make that work. instead of adding another set of similar invocations